### PR TITLE
feat: analytics project preview

### PR DESCRIPTION
**Pull request by rephus**
Closes: #2955

### Description:

Screenshot from 2022-08-24 13-26-07

### CODE REVIEW

File: packages/backend/src/services/ProjectService/ProjectService.ts [hunk 1]
```diff
@@ -127,6 +128,20 @@ export class ProjectService {
             user.organizationUuid,
             data,
         );
+        analytics.track({
```
Comment (rephus) [inline packages/backend/src/services/ProjectService/ProjectService.ts:131]: @owlas is `create` from line 148 also executed after the preview (if so, this project.created is redudant and we will be tracking the same event twice)

**Review by TuringLovesDeathMetal — APPROVED** [review]
nice! looks good to me :)

File: packages/backend/src/services/ProjectService/ProjectService.ts (hunk 1 above)
Comment (owlas) [reply packages/backend/src/services/ProjectService/ProjectService.ts:131]: nope it's a separate endpoint!

Comment (owlas) [reply packages/backend/src/services/ProjectService/ProjectService.ts:131]: merge away if you're happy

**Pull request merged**